### ISSUE TITLE
Not working on Linux ( Can discover but can't send messages / files )

### ISSUE THREAD

**Issue by melligeorgiou** [RobinLinus/snapdrop#430]
Hi,

Tested on PureOS (debian-based) and can discover the PCs and Macs on my network but I can't send or receive any message or file from those computers.

The PCs and Macs can send data between them just fine so it doesnt appear to be anything wrong with the network.

On Linux, I've tested Epiphany, Brave, Firefox, and Chrome. All browsers suffer with the same issue.

Is this a known problem? Does anyone have any fix for this?

**Comment by blabla-labALT** [comment]
Me too, I'm using Linux mint, Firefox.

**Comment by rzfang** [comment]
I have similar problem, can not work on Firefox, but can work on Chrome.

my env:
Linux Mint 20.3 Cinnamon
Firefox 99.0
Chrome 100.0.4896.127

**Comment by dodog** [comment]
Same issue on Manjaro Gnome 21.2.6, on Firefox 99.0 is does not work, but on Chromium 100.0 it does.
But sending files other way around from Firefox to iOS works well.

**Comment by rzfang** [comment]
New finding, **Reload** Linux Mint Firefox several times may get it workable.

I tried to send some photos from my iPhone Safari to my Linux Mint Firefox, After the both device pages were ready and saw each other, I picked photos from iPhone but nothing happened.
Then I tried reloading Linux Mint Firefox page, It was gone and back on iPhone Safari page, and I tried it again, this time iPhone could send the photos, and Linux Mint could recieve them.

**Comment by szymon420699** [comment]
yeah i have the same problem im on manjaro  on my pc and arch on  my laptop

**Comment by xundeenergie** [comment]
It does not work with Firefox on Android. Using another browser, i can send and receive file on android.
